code/obj/machinery/shield_generators/portable_shield_generator.dm

```diff
@@ -627,7 +635,7 @@ ADMIN_INTERACT_PROCS(/obj/machinery/shieldgenerator, proc/turn_on, proc/turn_off
 
 
 //sealab arrivalss
-/obj/machinery/door/var/obj/forcefield/energyshield/linked_forcefield = 0
+/obj/machinery/door/var/obj/forcefield/energyshield/linked_forcefield = null
```

Comment:
This is utterly cursed and shouldn't be here, I charge you with moving it to the `/obj/machinery/door` definition

Reply:
Done in dfad581